### ISSUE TITLE
Needs converting to .NET 5

### ISSUE THREAD

**Issue by VictorSago** [DeborahK/CSharpBP-Collections#6]
The current version of the application is completely unusable in new incarnation of the .NET. Trying to convert it manually is a nostarter, and all the tools I tried today are pretty useless.

**Comment by DeborahK** [comment]
Yes. I'm currently working with Pluralsight as they plan to update a majority of their C# catalog to .NET 5. We are meeting next week to discus and prioritize.

**Comment by DeborahK** [comment]
To make it work for now, your best bet is to create a new .NET 5 project using the current tooling, then copy the code files into that project. I don't think that the basic code was affected.

**Comment by VictorSago** [comment]
Thank you! I was thinking of doing just that. :)
